trigger/pubsub/util.go

```diff
@@ -51,10 +51,17 @@ func getClusterName(metadataEndpoint string) (string, error) {
 	return string(body), nil
 }
 
-// isGoogleContainerRegistry - we only care about gcr.io images,
+// isGoogleContainerRegistry - we only care about gcr.io and pkg.dev images,
```

Comment:
Because ArtifactRegistry appears to be the intended new normal, it'd make sense to change the function name to `isGoogleArtifactResgistry` here, on line 58, and in its implemented location

Reply:
I changed the function name.

Reply:
Thanks!